pebbles/views/namespaced_keyvalues.py

```diff
@@ -36,7 +36,7 @@ def get(self):
         if args.get('namespace'):
             namespaced_query = namespaced_query.filter_by(namespace=args.namespace)
         if args.get('key'):
-            namespaced_query = namespaced_query.filter_by(key=args.key)
+            namespaced_query = namespaced_query.filter(NamespacedKeyValue.key.like(("%s%%") % args.key))
```

Comment:
Will do. And actually it is supposed to be that the string should start with a certain prefix